timetracker/entries/urls.py

```diff
@@ -5,4 +5,7 @@
 
 urlpatterns = [
     url(r'^entries/$', views.entries, name='entries'),
+    url(r'^projects/$', views.projects, name='projects'),
+    url(r'^clients/$', views.clients, name='clients'),
+    url(r'^client/(?P<pk>\d*)/$', views.client_summary, name='client_summary')
```

Comment:
You probably want `(?P<pk>\d+)` (changed `*` to `+`). Otherwise a user can request `http://example.com/client//` which  is something else than `http://example.com/client/`. I.e. at least 1 digit (not just 0).